Alicerce/Sources/Logging/Formatters/LogItemLevelColorFormatter.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  LogItemLevelColorFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+protocol LogItemLevelColorFormatter {
```

Comment:
Why do we need a `LogItemLevelNameFormatter` and a `LogItemLevelColorFormatter`?

Shouldn't we simply have a `LogItemLevelFormatter` with a single `func stringForLevel()` ?
The `escape` and `reset` vars could be optional, if really necessary.